src/NonCumulative/Soundness/LogRel.agda

```diff
@@ -127,11 +127,8 @@ record ΛRel i j k Δ t IT OT (σ : Subst ) f
             (R$ : ∀ {a a′} → a ≈ a′ ∈ PERDef.El j (λ l<k → univ (ΠI≤ i≡maxjk l<k)) jA → Ctx → Exp → Typ → D → Set) : Set where
   field
     IT-rel : RI Δ (IT [ σ ])
-    ap-rel : Rs Δ s (IT [ σ ]) b → (b∈ : b ∈′ PERDef.El j (λ l<k → univ (ΠI≤ i≡maxjk l<k)) jA) → ΛKripke Δ (t [ σ ] $ s) (OT [ σ , s ∶ IT [ σ ] ↙ j ]) f b (R$ b∈)
-
-  flipped-ap-rel : (b∈ : b ∈′ PERDef.El j (λ l<k → univ (ΠI≤ i≡maxjk l<k)) jA) → ∀ {s} → Rs Δ s (IT [ σ ]) b → ΛKripke Δ (t [ σ ] $ s) (OT [ σ , s ∶ IT [ σ ] ↙ j ]) f b (R$ b∈)
```

Comment:
you might need this flipped variant

Reply:
I checked that there was only one place that used the flipped variant. I change the order of with in that proof so it's not used anymore there. I think it's safe to remove it.